includes/class-wc-payments-account.php

```diff
@@ -96,13 +104,12 @@ public function __construct(
 	public function init_hooks() {
 		// Add admin init hooks.
 		add_action( 'admin_init', [ $this, 'maybe_handle_onboarding' ] );
-		add_action( 'admin_init', [ $this, 'maybe_redirect_to_onboarding' ], 11 ); // Run this after the WC setup wizard and onboarding redirection logic.
-		add_action( 'admin_init', [ $this, 'maybe_redirect_to_wcpay_connect' ], 12 ); // Run this after the redirect to onboarding logic.
-		add_action( 'admin_init', [ $this, 'maybe_redirect_to_capital_offer' ] );
-		add_action( 'admin_init', [ $this, 'maybe_redirect_to_server_link' ] );
-		add_action( 'admin_init', [ $this, 'maybe_redirect_settings_to_connect_or_overview' ] );
-		add_action( 'admin_init', [ $this, 'maybe_redirect_onboarding_flow_to_overview' ] );
-		add_action( 'admin_init', [ $this, 'maybe_redirect_onboarding_flow_to_connect' ] );
+
+		add_action( 'admin_init', [ $this, 'maybe_redirect_after_plugin_activation' ], 11 ); // Run this after the WC setup wizard and onboarding redirection logic.
```

Comment:
`maybe_redirect_to_onboarding` -> `maybe_redirect_after_plugin_activation` to make it more readable